index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: The Morganton Scientific
+title: Morganton Scientific
 description: The Morganton Scientific, NCSSM's official journal of student STEM research
 ---
 
-Welcome to the The Morganton Scientific, NCSSM's official journal of student STEM research!
+Welcome to the *Morganton Scientific*, NCSSM's official journal of student STEM research!
```

Comment:
This is the content on the main page under the articles. You mentioned you might not want that?

Reply:
That's correct. I will remove this line and the following line completely if that will remove the text from under the articles.